cmd/cli/version/version.go

```diff
@@ -100,8 +101,12 @@ func (oV *VersionOptions) Run(ctx context.Context, cmd *cobra.Command) error {
 	if oV.ClientOnly {
 		versions.ClientVersion = version.Get()
 	} else {
+		// NB(forrest): since `GetAllVersions` is an API call - in the event the server is un-reachable
+		// we timeout after 3 seconds to avoid waiting on an unavailable server to return its version information.
+		vctx, cancel := context.WithTimeout(ctx, time.Second*3)
```

Comment:
what do we log or return to the user after the 3 seconds timeout?

Reply:
We only return the client version in that case:
```bash
> export BACALHAU_API_HOST=1.1.1.1
> bacalhau version
 CLIENT                UPDATE MESSAGE 
 v1.3.0-43-g9debb92e9                 

> export BACALHAU_API_HOST=bootstrap.production.bacalhau.org
> bacalhau version
 CLIENT                SERVER  LATEST  UPDATE MESSAGE 
 v1.3.0-43-g9debb92e9  v1.3.0  v1.3.0                 
```

The log message below this comment on line 112 isn't returned to the user on the CLI since we only log `Fatal` for CLI calls.